src/pages/home.tsx

```diff
@@ -0,0 +1,27 @@
+import { AddHeader } from 'components/header';
+import { MoviesList } from 'components/movies-list';
+import { ModalsContext } from 'modal-context';
+import React, { useContext } from 'react';
+
+export const Home = () => {
+  const {
+    toggleDelete,
+    toggleEdit,
+    toggleAdd,
+    setMovieDetails,
+    setLoadingMovieDetails,
+    setErrorMovieDetails,
+  } = useContext(ModalsContext);
+  return (
+    <>
+      <AddHeader hideAdd={toggleAdd} />
+      <MoviesList
+        setMovieDetails={setMovieDetails}
```

Comment:
Why do you even need to pass these props to the component if it is from context? You can get it directly in `MoviesList ` component. You have code duplication. Take a look at `MovieDetails` component